src/traces/scattergl/index.js

```diff
@@ -882,8 +893,8 @@ function plot(gd, subplot, cdata) {
     if(selectMode) {
         // create select2d
         if(!scene.select2d) {
-            // create scatter instance by cloning scatter2d
-            scene.select2d = createScatter(fullLayout._glcanvas.data()[1].regl, {clone: scene.scatter2d});
+            // create select scatter instance by cloning scatter2d
+            scene.select2d = createScatter(fullLayout._glcanvas.data()[1].regl);
```

Comment:
Since we reuse clustering, creating of new scatter is way faster, we don't need to clone data.

Reply:
sweet!

Reply:
... which makes switching from zoom/pan to select/lasso drag modes waaay faster. Thanks!